backend/spec/controllers/internal/companies/administrator/quickbooks_controller_spec.rb

```
@@ -0,0 +1,72 @@
+      integration # ensure integration exists
+    end
+
+    it "enqueues QuickbooksIntegrationSyncScheduleJob when authorized" do
+      expect(QuickbooksIntegrationSyncScheduleJob).to receive(:perform_async).with(company.id)
+
+      post :sync_integration, params: { company_id: company.id }
+
+      expect(response).to have_http_status(:ok)
+      expect(JSON.parse(response.body)).to eq({
+        "success" => true,
+        "message" => "QuickBooks sync initiated"
```

RuboCop (1.76.1)

[convention] Put a comma after the last item of a multiline hash.

(Style/TrailingCommaInHashLiteral)